docs/yul.rst

```diff
@@ -881,6 +885,16 @@ the ``dup`` and ``swap`` instructions as well as ``jump`` instructions, labels a
 | gaslimit()              |     | F | block gas limit of the current block                            |
 +-------------------------+-----+---+-----------------------------------------------------------------+
 
+.. _yul-call-return-area:
+
+The ``call*`` instructions use the ``out`` and ``outsize`` parameters to define an area in memory where
```

Comment:
Maybe we should turn this into a `Note`? And we could also move it below the following paragraph.